debian/control

```diff
@@ -120,7 +121,6 @@ Section: doc
 Architecture: all
 Multi-Arch: foreign
 Depends: ${misc:Depends},
-         ${sphinxdoc:Depends}
```

Comment:
NAK, this is incorrect packaging.

If this is adding unneeded dependencies, that needs to be fixed in `dh-sphinxdoc` (which is the source of `${sphinxdoc:Depends}`) and not worked around here.